src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -59,6 +61,33 @@ def dcos_browse(name, resource_group_name):
 
     return
 
+def dcos_install():
+    """
+    Downloads the dcos command line from Mesosphere
+    """
+    install_location = ""
+    file_url = ""
+
+    system = platform.system()
+    if system == 'Windows':
+        install_location = 'C:\Program Files\dcos.exe'
+        file_url = 'https://downloads.dcos.io/binaries/cli/windows/x86-64/dcos-1.8/dcos.exe'
+    elif system == 'Linux':
+        install_location = '/usr/local/bin/dcos'
+        file_url = 'https://downloads.dcos.io/binaries/cli/linux/x86-64/dcos-1.8/dcos'
+    elif system == 'Darwin':
+        install_location = '/usr/local/bin/dcos'
+        file_url = 'https://downloads.dcos.io/binaries/cli/darwin/x86-64/dcos-1.8/dcos'
+    else:
+        logger.error('unknown system: %s' % system)
+
+    user_location = input('dcos binary install location [%s]:' % install_location)
```

Comment:
Our convention is to not take user input interactively like this.
It makes it difficult to set the install_location with a script.
I'd suggest removing this and adding an optional argument `install_location` like I've described above.